cli/.js/src/main/scala/aqua/run/RunOpts.scala

```diff
@@ -33,6 +34,16 @@ object RunOpts extends Logging {
         "/dns4/kras-00.fluence.dev/tcp/19001/wss/p2p/12D3KooWR4cv1a8tv7pps4HH6wePNaK6gf1Hww5wcCMzeWxyNw51"
       )
 
+  val secretKeyOpt: Opts[Array[Byte]] =
+    Opts
+      .option[String]("sk", "Secret key in base64. Requests will be initialized with this key.", "s")
```

Comment:
I think "initialized with this key" is highly confusing. I think `--help` isn't a place to educate people on secret keys and peer ids, so let's just describe data format and leave it at it.

```suggestion
      .option[String]("sk", "Ed25519 32-byte secret key in base64", "s")
```